README.md

```diff
@@ -14,7 +14,6 @@ While there is no official support by HERE, you may still raise issues via GitHu
 
 ## Prerequisites
 This plugin is compatible with:
-- Scala 2.12
```

Comment:
I removed this as I believe it confuses users. The plugin is compatible with any Scala version in the project. The Scala 2.12 refers to the meta project Scala version which the user doesn't have control of, right?

Reply:
You are right